index.js

```diff
@@ -1,7 +1,32 @@
 const express = require('express');
 const path = require('path');
 const PORT = process.env.PORT || 3000;
+const ENV_DEV = 'development'
+const ENV_PROD = 'production';
+const ENV = process.env.NODE_ENV || ENV_DEV;
+
+const CONTENT_SECURITY_POLICY = [
+  "default-src *.scryfall.com",
+  "script-src *.scryfall.com 'unsafe-eval' 'unsafe-inline'",
+  "style-src *.scryfall.com 'unsafe-inline'",
```

Comment:
Allowing `unsafe-inline` cripples the value of CSP. Why does this app need those values?

Reply:
This should be resolved.